src/mlpack/methods/decision_tree/decision_tree.hpp

```diff
@@ -479,6 +503,7 @@ using DecisionStump = DecisionTree<FitnessFunction,
                                    ElemType,
                                    false>;
 
+typedef DecisionTree<InformationGain, BestBinaryNumericSplit, AllCategoricalSplit, AllDimensionSelect, double, true> ID3DecisionStump;
```

Comment:
Can we also add a little documentation here for this?  And it should wrap so it is 80 characters per line. :+1: